OrbitCore/TracepointEventBuffer.h

```diff
@@ -23,6 +23,16 @@ class TracepointEventBuffer {
   [[nodiscard]] const std::map<uint64_t, orbit_client_protos::TracepointEventInfo>&
   GetTracepointsOfThread(int32_t thread_id) const;
 
+  const std::deque<orbit_client_protos::TracepointEventInfo> GetTracepointEvents() {
+    std::deque<orbit_client_protos::TracepointEventInfo> tracepoint_events;
```

Comment:
You should use the mutex here. Also, as argued above, I would use a `ForEachTracepointEvent` method instead.

Reply:
Thank you so much for the advice! I will update the code!